plugins/com.google.cloud.tools.eclipse.integration.appengine/src/com/google/cloud/tools/eclipse/integration/appengine/NewMavenBasedAppEngineProjectWizardTest.java

```diff
@@ -48,40 +49,29 @@
 
   @Test
   public void testHelloWorld() throws Exception {
-    // appengine-skeleton-archetype still missing base index.html and HelloWorld.java
+    // appengine-standard-archetype still missing base index.html and HelloWorld.java
```

Comment:
I don't think this comment was true with the old archetype. Maybe not true either with the new archetype.